task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -4,6 +4,38 @@ namespace task_DEV_3
 {
     class Program
     {
+        static void InputOfArrayElements(int[] numberArray, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                numberArray[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void ExaminationNumberArray(int[] numberArray, int arrayLength)
```

Comment:
Change the name of the method to CheckNumbersForPrime(int[] numberArray, int arrayLength).
And it should return true or false.

Reply:
done